### ISSUE TITLE
Support update on all runtime

### ISSUE THREAD

**Issue by miton18** [CleverCloud/terraform-provider-clevercloud#98]
update is implemented on PHP,

we need to implement it on other runtime/addons

**Comment by TheCrabe** [comment]
A brief list to track progress

- [ ] addon
- [ ] cellar
- [ ] bucket
- [ ] docker
- [ ] java
- [ ] keycloak
- [ ] materiakv
- [ ] mongodb
- [ ] nodejs
- [ ] php
- [ ] postgresql
- [ ] python
- [ ] redis
- [ ] scala
- [ ] static

**Comment by mbrung** [comment]
I would expect the list to include Ruby, which is a supported CC runtime ;-)

**Comment by TheCrabe** [comment]
> I would expect the list to include Ruby, which is a supported CC runtime.

You're right. That's because Ruby is not implemented yet in our provider, and the list only includes runtimes that are already in it but without the update function.

I have just created the issue #133 to implement it.
